docs/getting-started/03-cli.md

```diff
@@ -90,6 +90,10 @@ curl -v http://127.0.0.1:4010/pets/123 -H "Prefer: dynamic=false"
 
 > **Remember:** you can combine `code`, `example` and `dynamic` parameters. By default, when the `code` parameter is not given, Prism will always try to fetch an example using the `exampleKey` from the 200 HTTP responses **only**. You'll need to use the `code` parameter alongside the `example` one for any specific example using a different HTTP status code than 200.
 
+#### Circular references
```

Comment:
Can we differentiate between types of self references that we support and infinitely circular references that would be impossible to reference? Maybe a few examples? I think we handle a fair few of JSON-schema-ref-parser sorts them out, but there’s some which we throw our hands up on right?

Reply:
I am not 100% sure that we can handle self references correctly either. Do you happen to have a document or something I can try with?

Reply:
I was talking about certain scenarios which are not truly circular. When a User has a manager which is a User that is often circular, but when that manager is optional then our tooling should be able to return null after X managers (1, 2, 3, 10) and bail. I'm referencing the discussions in https://github.com/stoplightio/prism/issues/891.

This wording makes it sound like we can't handle aaaaaaanything along these lines, and if thats the case then... well we need to get some of these easier scenarios working and have an issue to track it.